packages/gatsby/cache-dir/register-service-worker.js

```diff
@@ -1,10 +1,11 @@
-import emitter from "./emitter"
+import { apiRunner } from "./api-runner-browser"
 
 if (`serviceWorker` in navigator) {
   navigator.serviceWorker
     .register(`${__PATH_PREFIX__}/sw.js`)
     .then(function(reg) {
       reg.addEventListener(`updatefound`, () => {
+        apiRunner(`onServiceWorkerUpdateFound`)
```

Comment:
let's pass in `reg` as an option (though maybe call it just `serviceWorker` to all the API calls in case plugins want access to it.